app/src/organisms/Desktop/Camera/CameraControls/index.tsx

```diff
@@ -1,25 +1,52 @@
+import { useState } from 'react'
 import { useTranslation } from 'react-i18next'
 
-import { Modal, PrimaryButton, Slider } from '@opentrons/components'
+import {
+  Icon,
+  Modal,
+  PrimaryButton,
+  SecondaryButton,
+  Slider,
+} from '@opentrons/components'
+import { useCreateCameraImageSettings } from '@opentrons/react-api-client'
 
 import { TextOnlyButton } from '/app/atoms/buttons'
 import { Divider } from '/app/atoms/structure'
+import { useCameraSettingsValues } from '/app/local-resources/images/hooks/useCameraSettingsValues'
 
 import styles from './cameracontrols.module.css'
-import { useStubCameraSettingsValues } from './hooks/useStubCameraSettingsValues'
 import { PreviewSettings } from './PreviewSettings'
 import { ZoomSettings } from './ZoomSettings'
 
-import type { UseStubCameraSettingsValuesResult } from './hooks/useStubCameraSettingsValues'
+import type { UseCameraSettingsValuesResult } from '/app/local-resources/images/hooks/useCameraSettingsValues'
 
 export interface CameraControlsProps {
   onClose: () => void
 }
 
 export function CameraControls({ onClose }: CameraControlsProps): JSX.Element {
   const { t } = useTranslation('device_settings')
-
-  const settings = useStubCameraSettingsValues()
+  const settings = useCameraSettingsValues()
+  const [isLoading, setIsLoading] = useState(false)
+  const { createCameraImageSettings } = useCreateCameraImageSettings()
+  const handleSave = async (): Promise<void> => {
+    createCameraImageSettings(
```

Comment:
```suggestion
    setIsLoading(true)
    createCameraImageSettings(
```
Whoops, logic is slightly backwards here. We want to set the loading state to true on click and then to false  when the network request settles.